tools/matRad_getAlphaBetaCurves.m

```diff
@@ -0,0 +1,94 @@
+function [machine] = matRad_getAlphaBetaCurves(machine,varargin)
+% matRad alpha beta curve calculation tool
+%
+% call
+%   machine.data = matRad_getAlphaBetaCurves(machine.data,pln,cst)
+%
+% input
+%   machine.data:       matRad machine file to change
+%   varargin:       (optional) matRad cst struct and/or specify RBE modelName
+%                   to use for computation
+%
+% output
+%   machine.data:       updated machine file with alpha/beta curves
+%
+%
+% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%
+% Copyright 2021 the matRad development team.
+%
+% This file is part of the matRad project. It is subject to the license
+% terms in the LICENSE file found in the top-level directory of this
+% distribution and at https://github.com/e0404/matRad/LICENSES.txt. No part
+% of the matRad project, including this file, may be copied, modified,
+% propagated, or distributed except according to the terms contained in the
+% LICENSE file.
+%
+% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+matRad_cfg =  MatRad_Config.instance();
+
+if contains(machine.meta.radiationMode,{'car','hel'})
```

Comment:
why short forms car & hel? Also I would check explicitly for the ones you can compute (otherwise I could pass new oxygen data and it would try to do it). And why does helium with the LET model not work?

Reply:
helium works well with the analytical LET model, so I'm now checking if 'LEM' is parsed in or the mode is not helium or protons.